quickstarts/microsoft.compute/vm-simple-linux/main.bicep

```diff
@@ -159,8 +160,8 @@ resource vm 'Microsoft.Compute/virtualMachines@2021-11-01' = {
       }
       imageReference: {
         publisher: 'Canonical'
-        offer: 'UbuntuServer'
-        sku: ubuntuOSVersion
+        offer: '0001-com-ubuntu-server-jammy'
```

Comment:
this isn't going to work for any other value of the version parameter - to enable this we'd have to add a dictionary of imageReference....  so I'll close this PR, feel free to reopen if you want to enable that...